indra/assemblers/pybel_assembler.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import absolute_import, print_function, unicode_literals
+from builtins import dict, str
+from indra.statements import *
+from indra.databases import hgnc_client
+import logging
+import pybel
+import pybel.constants as pc
+from pybel.parser.modifiers import PmodParser
+
+# Python 2
+try:
+    basestring
+# Python 3
+except:
+    basestring = str
+
+logger = logging.getLogger('pybel_assembler')
+
+class PybelAssembler(object):
+
+    def __init__(self, stmts=None):
+        if stmts is None:
+            self.statements = []
+        else:
+            self.statements = stmts
+
+        self.model = None
+
+
+    def make_model(self, **kwargs):
+        self.model = pybel.BELGraph(**kwargs)
+        for stmt in self.statements:
+            # Convert phosphorylation statements into
+            # kin(p(agent)) => pmod(p(agent))
+            if isinstance(stmt, Phosphorylation):
+                # Define the enzyme node
+                (enz_node, enz_attr) = _get_agent_node(stmt.enz)
+                (sub_node, sub_attr) = _get_agent_node(stmt.sub)
+                sub_attr[pc.VARIANTS] = [
+                                    {pc.KIND: pc.PMOD,
+                                     pc.IDENTIFIER: {
+                                       pc.NAMESPACE: pc.BEL_DEFAULT_NAMESPACE,
+                                       pc.NAME: 'Ph'},
+                                     PmodParser.CODE: stmt.residue,
```

Comment:
Btw, are you using a *really* old version of PyBEL? I think I moved all of the constants out of the parser classes for easier reuse a long time ago.